### ISSUE TITLE
[MVP] Migrate to Lighthouse Benefits Intake API

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#69023]
## Issue Description
**User Story**

As a Pension team, **we want to kickoff engineering build efforts to migrate the Pension 527 form from `Central Mail API` to `Lighthouse Benefits Intake API`,** so that we can proceed with the goal align with the Benefits Portfolio Strategic Priorities - Focus on increasing stability, performance, and efficiency of our core products by eliminating the loss of VA.gov submissions of any kind.

---
## Tasks
- [x] Implement a Flipper for using Lighthouse API with 527EZ
- [x] Implement (or reuse) functionality similar to 686c/674 to use LH API
- [x] Use `uploads/validate_document` before submission

## Acceptance Criteria
- [x] New functionality for 527ez submission implemented and documented.
- [x] Lighthouse API should be primary submission path.
- [x] Feedback on failed submission.

**Comment by vbellissimo** [comment]
@wayne-weibel can I get your help refining this ticket, if we want to include it in this upcoming sprint? **Sprint 7 Plan: 11/08/2023—11/21/2023**

**Comment by wayne-weibel** [comment]
> @wayne-weibel can I get your help refining this ticket, if we want to include it in this upcoming sprint? **Sprint 7 Plan: 11/08/2023—11/21/2023**

@vbellissimo @mjknight50 I updated the comment, but please refine as needed

**Comment by wayne-weibel** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNy9pQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--95b51c312860361d66ad713451d051362afffc82/image.png)

**Comment by wayne-weibel** [comment]
will need to iterate with another ticket/job to fully confirm uploaded documents
https://dsva.slack.com/archives/C053U7BUT27/p1700233242312199

**Comment by mjknight50** [comment]
The decision in our daily was that this ticket can be closed once the PR is merged.  We will open a new ticket for the follow on work related to polling.
